qiime/plugin/testing.py

```diff
@@ -84,3 +90,33 @@ def assertSemanticTypeRegisteredToFormat(self, semantic_type, exp_format):
             obs_format, exp_format,
             "Expected semantic type %r to be registered to format %r, not %r."
             % (semantic_type, exp_format, obs_format))
+
+    def transform_format(self, source_format, target, **kwargs):
```

Comment:
Now that makes more sense, I got thrown off by review having the `kwargs` being in code highlighting over just making them "keywords". I'll get these all tweaked today, my head was apparently out of it for a few of these.